code/index.html

```diff
@@ -1,30 +1,178 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css"/>
+    <title>Project 3: Business site</title>
+    <link rel="icon" href="./videos/icon-fish.png" type="image/png" sizes="16x16">
+</head>
+<body>
+    
+    <header class="hero-section">
+      <div class="hero-video">
+        <video src="./videos/video_large.mp4" autoplay muted loop></video>
+        <img src="./videos/poster1.png" title="Your browser does not support the video tag.">
       </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="hero-text">
+        <h1>Two Fish</h1>
+        <h2>Contest</h2>
+        <h3>Win a scuba-diving trip for you and your special-one!</h3>
+        <a href="#intro" class="btn">Register now</a>
       </div>
+    </header>
+
+    <section id="intro">
+      <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+      Vitae error ratione a dolorem odio sit omnis, iure est optio sunt nulla.
+      Vitae error ratione a dolorem.</p>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <section class= flex-parent>    
```

Comment:
Don't forget quotation marks, and there should be no space after the equal sign!

Reply:
ok